feats/utils.py

```diff
@@ -0,0 +1,34 @@
+from collections import defaultdict
+from typing import Dict
+
+from .meta import Implementation
+
+
+def fn_to_implementations(fn):
```

Comment:
I think by putting this in a separate you've made testing and usage a bit more complicated. If this was `Definition.from_function(fn)` and `Definition.from_object(obj)`, it would be natural to keep the existing tests, and would simplify the actual usage of the functions (there wouldn't need to be __doc__ handling in the app)

Reply:
I think this is important enough to change before merging. Will make this update and re-request a ✅